keyboards/ziptyze/lets_split_v3/config.h

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2023 ziptyze
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN GP16
+#define SERIAL_USART_RX_PIN GP17
+#define SERIAL_USART_PIN_SWAP
+
+#define SPLIT_TRANSPORT_MIRROR
+#define RGB_MATRIX_LED_COUNT 126
+#define RGB_MATRIX_KEYPRESSES // reacts to keypresses
+#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
```

Comment:
KEYPRESSES and FRAMEBUFFER_EFFECTS are enabled when an `rgbmatrix.animation` is enabled 
```suggestion
#define RGB_MATRIX_LED_COUNT 126
#define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
```